### PR TITLE
Improve confluence plugin response

### PR DESCRIPTION
**Pull request by krzyurb**
Add author, space name and creation date to each confluence result

### CODE REVIEW

File: src/confluence/confluence.service.js [hunk 1]
```diff
@@ -21,10 +22,27 @@ class ConfluenceService {
...
       const url = `${this.baseUrl}/wiki${_links.webui}`;
+      const authorUrl = `${this.baseUrl}/people/${history.createdBy.accountId}`;
+      const authorIconUrl = `${this.baseUrl}${history.createdBy.profilePicture.path}`;
 
-      return { title, title_link: url, color: randomColor() };
+      return {
+        title,
+        title_link: url,
+        footer: space.name,
+        author_name: history.createdBy.displayName,
```
Comment (Getriax) [inline src/confluence/confluence.service.js]: So I have noticed that the user can be undefined. In that case, I am getting 
`Unknown User (fa2a....534sfd2)`

Moreover looking at the response JSON I saw that `createdBy.displayName` could be equal to 
`Unknown User (fa2a....534sfd2)` while the `createdBy.username` would point to a user.

What do you think about adjusting that a little? 😄

Comment (krzyurb) [reply src/confluence/confluence.service.js]: Goooood point!

**Comment by krzyurb** [comment]
@Getriax fixed

File: src/confluence/confluence.service.js [hunk 2]
```diff
@@ -20,28 +20,33 @@ class ConfluenceService {
...
+  authorData(author) {
+    if (author.accountId) {
+      return {
+        author_name: author.displayName,
+        author_link: `${this.baseUrl}/people/${author.accountId}`,
+        author_icon: `${this.baseUrl}${author.profilePicture.path}`,
+      };
+    }
+
+    return {};
```
Comment (Getriax) [inline src/confluence/confluence.service.js]: Could you test that case?

**Pull request merged**